docs/admin/kubelet.md

```diff
@@ -80,7 +81,7 @@ kubelet
       --container-runtime="docker": The container runtime to use. Possible values: 'docker', 'rkt'. Default: 'docker'.
       --containerized[=false]: Experimental support for running kubelet in a container.  Intended for testing. [default=false]
       --cpu-cfs-quota[=true]: Enable CPU CFS quota enforcement for containers that specify CPU limits
-      --docker-endpoint="": If non-empty, use this for the docker endpoint to communicate with
+      --docker-endpoint="unix:///var/run/docker.sock": If non-empty, use this for the docker endpoint to communicate with
```

Comment:
Given that we are defaulting now, should we include the phrase ` If non-empty`? Even the defaults are explicit right?